ui/pages/Project/reducers.js

```diff
@@ -221,7 +221,7 @@ export const loadMmeMatches = (submissionGuid, search) => (dispatch, getState) =
 
 export const loadRnaSeqData = individualGuid => (dispatch, getState) => {
   const data = getState().rnaSeqDataByIndividual[individualGuid]
-  if (!data) {
+  if (!data || Object.values(data).every(({ isSignificant }) => isSignificant)) {
```

Comment:
It is a little confusing why it need to load data when every individual is significant. Add a comment to the code?